pytest_reana/fixtures.py

```diff
@@ -48,9 +49,13 @@ def test_dir_exists(tmp_shared_volume_path):
             assert os.path.exists(path)
 
     """
-    temp_path = str(tmpdir_factory.mktemp("reana"))
-    yield temp_path
-    shutil.rmtree(temp_path)
+    shared_volume_path = os.getenv("SHARED_VOLUME_PATH", "")
```

Comment:
See how this is being used in https://github.com/reanahub/reana-workflow-controller/pull/338